src/components/DonationForm/DonationForm.jsx

```diff
@@ -38,6 +38,7 @@
 import ItemInfo from '../ItemInfo/ItemInfo';
 import { STATUSES } from '../../utils/config';
 import DonationImageModal from '../DonationImageModal/DonationImageModal';
+import DonorFooter from '../DonorFooter/DonorFooter';
```

Comment:
🚫 **[eslint]** <[import/no-named-as-default-member](https://github.com/import-js/eslint-plugin-import/blob/v2.27.5/docs/rules/no-named-as-default-member.md)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Parse errors in imported module '../DonorFooter/DonorFooter': Unexpected token "
    >
      <HStack columnGap={2}>
        <Image boxSize={55} src={PNPlogo} display={{ base: 'none', md: 'block' }} />
        <Link
          href=" (16:25)